app/config/settings_prod.py

```diff
@@ -24,6 +24,12 @@
 # the setup
 APP_ENV = os.environ.get('APP_ENV', 'local')
 
+# Determine the APP version
+try:
+    from .version import APP_VERSION  # pylint: disable=unused-import
+except ModuleNotFoundError:
+    APP_VERSION = 'No version info'
```

Comment:
Maybe add a comment why this could happen and we are ok with it.

Reply:
It actually shouldn't happen, I changed it